pkg/pod/status.go

```diff
@@ -114,7 +104,7 @@ func MakeTaskRunStatus(logger *zap.SugaredLogger, tr v1beta1.TaskRun, pod *corev
 		trs.SetCondition(&apis.Condition{
```

Comment:
Should we create MarkResource* routine for these conditions and use those?

Reply:
We might do so to reduce boilerplate code, but I would not do it in this PR - https://github.com/tektoncd/pipeline/issues/2770